package.json

```diff
@@ -405,6 +406,9 @@
     "eslint-plugin-react": "7.29.4",
     "eslint-plugin-react-hooks": "4.4.0",
     "fetch-mock": "9.10.7",
+    "gitbook-plugin-hints": "^1.0.2",
+    "honkit": "^4.0.4",
```

Comment:
Why are you adding honkit packages? Our gitbook documentation is hosted on gitbook.com.

Reply:
Tried HonKit, didn't manage a full migration, forgot to revert all